styles/cards.less

```diff
@@ -0,0 +1,22 @@
+.amaRomach-card {
+    flex: 1 !important;
+    flex-basis: 0 !important;
```

Comment:
explain

Reply:
It means the basis of the flexbox so zero will collapse the element to its smallest possible width given the content